packages/docs/pages/Switch/SwitchPage.tsx

```diff
@@ -9,21 +9,21 @@ const SwitchPage = () => (
     <H0>Switch</H0>
 
     <Text>
-      Switches are a stylized <Code>input[type="checkbox"]</Code> with controllable checked/unchecked states.
+      Switches are a stylized <Code>input[type="checkbox"]</Code> with controllable checked/unchecked states. Switches
+      are intended to be used for immediate toggle actions and are therefore not intended to be used in forms.
     </Text>
 
     <CodePreview>
       {/* jsx-to-string:start */}
       {function Example() {
         const [checked, setChecked] = useState(false);
         const handleChange = () => setChecked(!checked);
+        const boxColor = checked ? 'white' : 'secondary20';
 
         return (
-          <Form>
-            <FormGroup>
-              <Switch checked={checked} onChange={handleChange} />
-            </FormGroup>
-          </Form>
+          <Box display="inline-block" padding="xLarge" backgroundColor={boxColor}>
```

Comment:
Let's just remove this altogether.
```suggestion
          <Box padding="xLarge">
```

Reply:
I understand removing the `display` prop (I forgot to take it out after some experimentation), but the intent of the box color was to show the `Switch` as an immediate state toggle. Do you have any suggestions of how that can be made visually clear/explicit, or do you think the description alone will be enough for anyone to understand the intent?

Reply:
I think the description along should be :gucci: enough. We just want to make sure we have it documented in text that the intent on using this component is not for forms.